### PR TITLE
Fix case sensitivity for all include files

### PR DESCRIPTION
**Pull request by galanCA**
## Description

Fixes include case sensitivity for GCC and Clang. This pull request should only change the problems related to case-sensitivity in include files, and cmakelist. A branch was created from this branch to change the name of all of the files to reflect lower case. This pull request should not have an impact in Windows CSE behavior.

## Author Progress Checklist:

- [x] Open draft pull request
    - [x] Make title clearly understandable in a standalone change log context
    - [x] Assign yourself the issue
    - [x] Add at least one label (enhancement, bug, or maintenance)
    - [ ] Link the issue(s) addressed by this PR (under "Development" in the sidebar menu) 
- [x] Make code changes (if you haven't already)
- [ ] Self-review of code
    - [x] My code follows the style guidelines of this project
    - [ ] I have added comments to my code, particularly in hard-to-understand areas
    - [x] I have only committed the necessary changes for this fix or feature
    - [ ] I have made corresponding changes to the documentation
    - [x] My changes generate no new warnings
    - [x] I have ensured that my fix is effective or that my feature works as intended by:
        - [ ] exercising the code changes in the test framework, and/or
        - [x] manually verifying the changes (as explained in the the pull request description above)
    - [x] My changes pass all local tests
    - [x] My changes successfully passes CI checks
    - [ ] Add any unit test for proof and documentation.
    - [x] Merge in main branch and address resulting conflicts and/or test failures.
- [ ] Move pull request out of draft mode and assign reviewers
- [ ] Iterate with reviewers until all changes are approved
    - [ ] Make changes in response to reviewer comments
    - [ ] Merge in main branch and address resulting conflicts and/or test failures.
    - [ ] Re-request review in GitHub

## Reviewer Checklist:

 - [x] Read the pull request description
 - [x] Perform a code review on GitHub
 - [x] Confirm all CI checks pass and there are no build warnings
 - [ ] Pull, build, and run automated tests locally
 - [ ] Perform manual tests of the fix or feature locally
 - [x] Add any review comments, if applicable
 - [x] Submit review in GitHub as either
     - [ ] Request changes, or
     - [x] Approve
 - [x] Iterate with author until all changes are approved

### CODE REVIEW

**Comment by nealkruis** [comment]
@galanCA can you merge master?

**Comment by galanCA** [comment]
@nealkruis master is merged.

**Comment by nealkruis** [comment]
@chipbarnaby any heartburn here?

**Review by chipbarnaby — COMMENTED** [review]
Questions:
1.  Could all these name be lower case?  Is the issue consistency with how the files are actually named in the file system?  Given a choice, I would use all lower case -- that seems more in the C/C++ tradition.
2. Assuming we're going for consistency, why does CSE.h have a lower-case h?
3. The \<filename\> form should be changed to "filename" given all files are being visited.  Back in the day, some of the include files were not in the src directory.  Now they are, so "filename" is appropriate.  We now have the kludge that adds "." as an include search location, so \<filename\> will be found in the current directory.

We should discuss and decide policy on these issues.

**Comment by nealkruis** [comment]
@galanCA I agree with your conclusions. Let move forward with those changes.

File: src/cnhp.cpp [hunk 1]
```diff
@@ -10,19 +10,19 @@
...
+#include "cnglob.h"
 
-#include <ancrec.h>	// record: base class for rccn.h classes
-#include <rccn.h>	// HEATPLANT BOILER including mbr fcn decls NHPSTAGES HPSTAGESZ
-#include <msghans.h>	// MH_R1350
+#include "ancrec.h"	// record: base class for rccn.h classes
+#include "rccn.h"	// HEATPLANT BOILER including mbr fcn decls NHPSTAGES HPSTAGESZ
+#include "msghans.h"	// MH_R1350
 
-//#include <pp.h>   	//
```
Comment (tanaya-bigladder) [inline src/cnhp.cpp:19]: If these includes have been commented for years, I'd just take them out.

**Review by tanaya-bigladder — CHANGES_REQUESTED** [review]

File: src/cnhp.cpp (hunk 1 above)
Comment (galanCA) [reply src/cnhp.cpp:19]: Done, it is not being shown as outdated.

**Review by nealkruis — APPROVED** [review]
@chipbarnaby on to you for review.

**Comment by nealkruis** [comment]
Although, @tanaya-bigladder still hasn't approved since the last changes.

File: src/cnhp.cpp (hunk 1 above)
Comment (tanaya-bigladder) [reply src/cnhp.cpp:19]: Huh, that's so weird.

**Review by tanaya-bigladder — APPROVED** [review]

**Comment by chipbarnaby** [comment]
I added some comments.

**Comment by nealkruis** [comment]
This is already merged. If there are still issues, please make another GH issue.

**Pull request merged**